modules/qr-code-generator/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "homepage": "https://github.com/nayuki/QR-Code-generator",
+  "maintainers": [
+      {
+          "email": "shaban_ziad@hotmail.com",
+          "github": "shabanzd",
+          "name": "Ziad Shaban"
+      }
+  ],
+  "repository": [
+      "https://github.com/nayuki/QR-Code-generator"
```

Comment:
This should be: `"github:nayuki/QR-Code-generator"` so that it does better check for github, can you please send another PR to fix it?

Reply:
Sent https://github.com/bazelbuild/bazel-central-registry/pull/3718